azext_iot/iothub/_help.py

```diff
@@ -309,3 +309,43 @@ def load_iothub_help():
         type: command
         short-summary: Upload a local file as a device to a pre-configured blob storage container.
     """
+
+    helps["iot edge hierarchy"] = """
+        type: group
+        short-summary: Commands to deploy and manage nested IoT Edge device hierarchies.
+    """
+
+    helps["iot edge hierarchy create"] = """
+        type: command
+        short-summary: |
+          Deploy nested edge hierarchies to an IoT Hub.
+          This can be done with inline device arguments or by supplying a nested edge configuration file in YAML or JSON format.
```

Comment:
`long-summary` can be used where there is ample enough information to provide in help. Short-summary for each command will be shown in its entirety when showing help for a command group.